test/core/messages.integration.test.ts

```diff
@@ -288,20 +308,26 @@ describe('messages integration', () => {
     );
 
     // Do a history request to get the update message
+    await new Promise((resolve) => setTimeout(resolve, 3000)); // wait for persistence - this will not be necessary in the future
     const history = await room.messages.get({ limit: 3, direction: 'forwards' });
 
     expect(history.items).toEqual([
       expect.objectContaining({
         text: 'Hello test!',
         clientId: chat.clientId,
         serial: updatedMessage1.serial,
-        updatedAt: updatedMessage1.updatedAt,
-        updatedBy: chat.clientId,
         createdAt: message1.createdAt,
-        createdBy: message1.clientId,
+        timestamp: updatedMessage1.updatedAt,
+        action: ChatMessageActions.MessageUpdate,
       }),
     ]);
 
+    // test shorthand getters
+    expect(history.items[0]?.isUpdated).toBe(true);
+    expect(history.items[0]?.isDeleted).toBe(false);
+    expect(history.items[0]?.updatedAt).toEqual(updatedMessage1.updatedAt);
+    // expect(history.items[0]?.updatedBy).toEqual(updatedMessage1.updatedBy);
```

Comment:
Same here, should this be commented out or not? Can we uncomment or remove it please?

Reply:
added comment to both of these, a todo to uncomment when operation is returned from the endpoint